src/store/auth.js

```diff
@@ -5,11 +5,13 @@ import { setCookie } from 'components/Shared/Cookies';
 import { setTokenOnHeader } from 'api/logined';
 import * as authAPI from 'api';
 
-const [LOGIN, LOGIN_SUCESS, LOGIN_FAILURE] = createRequestSagaActionTypes('auth/LOGIN');
-const [REFRESH, REFRESH_SUCESS, REFRESH_FAILURE] = createRequestSagaActionTypes('auth/REFRESH');
-const [SIGNUP, SIGNUP_SUCESS, SIGNUP_FALIURE] = createRequestSagaActionTypes('auth/SIGNUP');
+const [LOGIN, LOGIN_SUCCESS, LOGIN_FAILURE] = createRequestSagaActionTypes('auth/LOGIN');
+const [REFRESH, REFRESH_SUCCESS, REFRESH_FAILURE] = createRequestSagaActionTypes('auth/REFRESH');
+const [SIGNUP, SIGNUP_SUCCESS, SIGNUP_FALIURE] = createRequestSagaActionTypes('auth/SIGNUP');
+const [NON_MEMBER, NON_MEMBER_SUCCESS, NON_MEMBER_FALIURE] = createRequestSagaActionTypes('auth/NON_MEMBER');

```

Comment:
취향 차인데...`NON_MEMBER` 보단 `GUEST` 가 가독성 면에서 더 도움이 될 것 같아요  
```suggestion
const [GUEST_LOGIN, GUEST_LOGIN_SUCCESS, GUEST_LOGIN_FAIL] = createRequestSagaActionTypes('auth/NON_MEMBER');

```

Reply:
5769597 GUEST가 좀더 어울리는 것 같습니다